packages/core/src/libraries/ogcio-constants.ts

```diff
@@ -12,3 +12,6 @@ export const OGCIO_ORGANIZATION_ROLES = {
 export const OGCIO_ROLES = {
   BB_CITIZEN: 'bb-citizen',
 };
+
+export const OGCIO_MY_GOV_ID_IDENTITY = 'MyGovId (MyGovId connector)';
+export const OGCIO_ENTRA_ID_ENTITY = 'OGCIO EntraID';
```

Comment:
Small suggestion to make it consistent with the other const.

```suggestion
export const OGCIO_ENTRA_ID_IDENTITY = 'OGCIO EntraID';
```

Reply:
Oh thank you! Sure, that is supposed to be identity, it is just a typo. Thanks for the fix!